client/src/components/Common/UI/Icon.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import { StyledProps } from 'types/app';
+
+const IconName: IconName = {
+  Google: 'images/googleLogo.svg',
+  Kakao: 'images/kakaoLogo.svg',
+  Naver: 'images/NaverLogo.png',
+};
```

Comment:
type SocialIconNames = 'Google' | 'Kakao' | 'Naver';

type IconName = {
  [key in SocialIconNames]: string | undefined;
};